README.md

```diff
@@ -25,13 +25,13 @@ For building the Nighthawk test server, see [here](source/server/README.md).
 ### Test it
 
 ```bash
-bazel test -c dbg //test:nighthawk_test
+bazel test -c dbg //test/... //integration/...
```

Comment:
Can we move integration underneath test? It seems cleaner to have all tests in a single tree?

Reply:
That is where they lived at first, but it caused problems with the coverage test:
https://circleci.com/gh/envoyproxy/nighthawk/1193?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Can you update to use the new LLVM-style coverage script? See https://github.com/envoyproxy/envoy/pull/7552. This might not fix it, but it makes coverage much faster and easier to debug :)